README.md

```diff
@@ -175,20 +186,23 @@ make undeploy
 ```sh
 make uninstall
 ```
+
 ### Deploy with OLM
-2. Build and push bundle and index images to an image registry
+
+7. Build and push bundle and index images to an image registry. For podman, use `make podman-build-and-push-bundle-images`
```

Comment:
order is messed up here ? with OLM once cluster is up we build and push then deploy nothing else was needed ?

Reply:
Don't I need to build the daemon and controller image?  (at least I need to build and push the daemon image, because that's not provided in the bundle? Unless something is messed up in the way I deploy)

Reply:
right OLM assumes images are available